File: CSharp.Functions/Common/Entities/FirmDetails/OwnersAndControllers/FinancialStatus.cs
namespace Common.Entities
{
    public class FinancialStatus
    {
        public string? AnnualIncome { get; set; }
        public string? TotalAssets { get; set; }
        public string? TotalLiabilities { get; set; }
        public Money TotalAmountToActAsGuarantor { get; set; } = new Money();
        public string? SourceOfIncome { get; set; }
        public string? AdditionalInformation { get; set; }
    }
}

File: vue.jsonforms/src/entities/owners-and-controllers/FinancialStatus.ts
import {Money} from "@/entities/Money";

export class FinancialStatus {
    public annualIncome: string | undefined;
    public sourceOfIncome: string = "";
    public totalAssets: string | undefined;
    public totalLiabilities: string | undefined;
    public totalAmountToActAsGuarantor: Money = new Money();
    public additionalInformation: string = "";
}